README.md

```diff
@@ -7,7 +7,7 @@
 </p>
 
 **Stable**
-[![Build Status](https://travis-ci.com/yiisoft/yii-docker.svg?branch=master)](https://travis-ci.com/yiisoft/yii-docker)
+tbd
```

Comment:
What does it mean?

Reply:
to be done

when build on GH passes

Reply:
<img src="https://api.travis-ci.com/yiisoft/yii-docker.svg?branch=master">

Reply:
Could be applied now, I think.

Reply:
added